stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,577 @@
+export default {
+    title: 'Components/Feed List and Item',
+    parameters: {
+        description: `The feed list displays feed list items that individually consist of a username, a written entry and an optional image. There is also a separate byline below that can contain a time stamp or an attribute in the form of free text.
+       
+        
+##Usage        
+**Use the feed list if:**
+
+-        You expect multiple instances, such as notes or feed entries.
+-        The items in the feed list are homogenous.
+-        A user needs to input small amounts of text without formatting.
+
+**Do not use feed list if:**
+
+-       The user needs to format the text (rich text editor).
+-       You need only a single text box instance. Instead, use **Text Area**.
+-       You want to display empty fields for the user to input feed entries. Instead, use **Feed Input**.
+        `,
+        tags: ['f3', 'a11y', 'theme'],
+        components: ['action-sheet', 'feed-list', 'avatar', 'icon', 'button', 'menu', 'popover', 'link']
+    }
+};
+
+export const standard = () => `
+    <ul class="fd-feed-list" aria-label="Feed List Item default example">
+        <li class="fd-feed-list__body">
+            <div 
+                class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list__thumb"
+                style="background-image: url(http://lorempixel.com/460/620/people/7/)" 

```

Comment:
lorempixel.com seems to be offline. How about open-source https://avatars.dicebear.com/
```suggestion
                style="background-image: url(https://avatars.dicebear.com/api/human/fe.svg?m=6&b=%23f5f0f0&mood[]=surprised)" 
```

![image](https://user-images.githubusercontent.com/43764832/96296300-e9b57000-0fa3-11eb-8b3c-cc6441a70cd7.png)

Reply:
Yea, Good idea